File: features/promo/tokopoints/src/main/java/com/tokopedia/tokopoints/view/model/section/LayoutCategoryAttr.java

package com.tokopedia.tokopoints.view.model.section;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LayoutCategoryAttr {

    @SerializedName("categoryTokopointsList")
    @Expose
    private List<CategoryTokopointsList> categoryTokopointsList = null;

    public List<CategoryTokopointsList> getCategoryTokopointsList() {
        return categoryTokopointsList;
    }

    public void setCategoryTokopointsList(List<CategoryTokopointsList> categoryTokopointsList) {
        this.categoryTokopointsList = categoryTokopointsList;
    }

}

File: features/promo/tokopoints/src/main/java/com/tokopedia/tokopoints/view/model/section/LayoutCategoryAttr.kt
package com.tokopedia.tokopoints.view.model.section

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class LayoutCategoryAttr(
    @SerializedName("categoryTokopointsList")
    @Expose
    var categoryTokopointsList: List<CategoryTokopointsList> = listOf()
)
